tests/test_conv.c

```diff
@@ -90,13 +90,22 @@ static inline int clz32(unsigned int a)
 #endif
 }
 
+/* WARNING: undefined if a = 0 */
 /* WARNING: undefined if a = 0 */
 static inline int clz64(uint64_t a)
 {
-#if defined(_MSC_VER) && !defined(__clang__)
+#if defined(_MSC_VER)&& defined(_WIN64) && !defined(__clang__)
     unsigned long index;
     _BitScanReverse64(&index, a);
     return 63 - index;
+#elif(_MSC_VER)
+unsigned long i = 0, hi = 0;
+    if (_BitScanReverse(&hi, a>> 32))
+    {
+        return (int)hi + 32;
+    }
+    _BitScanReverse(&i, (uint32_t)a);
+    return (int)i;
```

Comment:
Same problems as above and duplicate comment line.

Reply:
If  `__lzcnt`, `__lzcnt32` or `__lzcnt64` are available all current Windows targets, using these intrinsics would allow for a simpler implementation.

Reply:
@startewho: Why not use the version quoted by @mekhontsev in the issue #430:
```
static inline int clz64(uint64_t a)
{
#if defined(_MSC_VER) && !defined(__clang__)
#if INTPTR_MAX == INT64_MAX
    unsigned long index;
    _BitScanReverse64(&index, a);
    return 63 - index;
#else
    if (a >> 32)
        return clz32((unsigned)(a >> 32));
    else
        return clz32((unsigned)a) + 32;
#endif	
#else
    return __builtin_clzll(a);
#endif
}
```

Reply:
> If `__lzcnt`, `__lzcnt32` or `__lzcnt64` are available all current Windows targets, using these intrinsics would allow for a simpler implementation.

https://learn.microsoft.com/zh-cn/cpp/intrinsics/lzcnt16-lzcnt-lzcnt64?view=msvc-170
it seems only support amd and intel cpu.